cibyl/exceptions/source.py

```diff
@@ -14,13 +14,14 @@
 #    under the License.
 """
 
+from cibyl.utils.colors import Colors
 
 class TooManyValidSources(Exception):
     """Exception for trying to use multiple supported sources."""
 
     def __init__(self, system):
         self.system = system
-        self.message = f"""You have more than one source defined for the system
+        self.message = Colors.yellow(f"""You have more than one source defined for the system
```

Comment:
Some of the exceptions are in yellow while some are in red. We probably want to stick to one color and that might be red since the meaning of getting an exception is stopping application execution